Assets/HoloToolkit/UI/Shaders/3DTextShader.shader

```diff
@@ -48,29 +50,39 @@
             CGPROGRAM
                 #pragma vertex vert
                 #pragma fragment frag
+				#pragma multi_compile_instancing
                 #include "UnityCG.cginc"
 
                 struct appdata_t
                 {
                     float4 vertex : POSITION;
                     half4 color : COLOR;
                     float2 texcoord : TEXCOORD0;
+					UNITY_VERTEX_INPUT_INSTANCE_ID
                 };
 
                 struct v2f
                 {
                     float4 vertex : POSITION;
                     half4 color : COLOR;
                     float2 texcoord : TEXCOORD0;
+					UNITY_VERTEX_INPUT_INSTANCE_ID
+					UNITY_VERTEX_OUTPUT_STEREO
+
                 };
 
                 sampler2D _MainTex;
                 float4 _MainTex_ST;
-                fixed4 _Color;
+				UNITY_INSTANCING_CBUFFER_START(Props)
+					UNITY_DEFINE_INSTANCED_PROP(fixed4, _Color)
```

Comment:
What's with the strange space formatting?

Reply:
The formatting keeps getting funky, I will fix.